pkg/compose/create.go

```diff
@@ -1124,13 +1124,26 @@ func (s *composeService) resolveExternalNetwork(ctx context.Context, n *types.Ne
 	networks, err := s.apiClient().NetworkList(ctx, moby.NetworkListOptions{
 		Filters: filters.NewArgs(filters.Arg("name", n.Name)),
 	})
+
 	if err != nil {
 		return err
 	}
 
+	if len(networks) == 0 {
+		networks, err = s.apiClient().NetworkList(ctx, moby.NetworkListOptions{
```

Comment:
here we assume n.Name is actually network ID, then `NetworkInspect` would be a better API to use

Reply:
Thank you!

Reply:
This has been changed.